2022q2/ci.adoc

```diff
@@ -0,0 +1,53 @@
+
+Links: +
+link:https://ci.FreeBSD.org[FreeBSD Jenkins Instance] URL: link:https://ci.FreeBSD.org[https://ci.FreeBSD.org] +
+link:https://artifact.ci.FreeBSD.org[FreeBSD CI artifact archive] URL: link:https://artifact.ci.FreeBSD.org[https://artifact.ci.FreeBSD.org] +
+link:https://wiki.freebsd.org/Jenkins[FreeBSD Jenkins wiki] URL: link:https://wiki.freebsd.org/Jenkins[https://wiki.freebsd.org/Jenkins] +
+link:https://wiki.freebsd.org/HostedCI[Hosted CI wiki] URL: link:https://wiki.freebsd.org/HostedCI[https://wiki.freebsd.org/HostedCI] +
+link:https://wiki.freebsd.org/3rdPartySoftwareCI[3rd Party Software CI] URL: link:https://wiki.freebsd.org/3rdPartySoftwareCI[https://wiki.freebsd.org/3rdPartySoftwareCI] +
+link:https://preview.tinyurl.com/y9maauwg[Tickets related to freebsd-testing@] URL: link:https://preview.tinyurl.com/y9maauwg[https://preview.tinyurl.com/y9maauwg] +
+link:https://github.com/freebsd/freebsd-ci[FreeBSD CI Repository] URL: link:https://github.com/freebsd/freebsd-ci[https://github.com/freebsd/freebsd-ci] +
+link:https://lists.freebsd.org/subscription/dev-ci[dev-ci Mailing List] URL: link:https://lists.freebsd.org/subscription/dev-ci[https://lists.freebsd.org/subscription/dev-ci]
+
+Contact: Jenkins Admin <jenkins-admin@FreeBSD.org> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org> +
+Contact: link:https://lists.FreeBSD.org/mailman/listinfo/freebsd-testing[freebsd-testing Mailing List] +
+Contact: IRC #freebsd-ci channel on EFNet
+
+The FreeBSD CI team maintains the continuous integration system of the FreeBSD project.
+The CI system checks the committed changes can be successfully built, then performs various tests and analysis over the newly built results.
+The artifacts from those builds are archived in the link:https://artifact.ci.FreeBSD.org[artifact server] for further testing and debugging needs.
+The CI team members examine the failing builds and unstable tests and work with the experts in that area to fix the code or adjust test infrastructure.
+
+During the second quarter of 2022, we continued working with the contributors and developers in the project to fulfil their testing needs and also keep collaborating with external projects and companies to improve their products and FreeBSD.
+
+Important completed tasks:
+
+* Fixed the hardware failure issue of the CI cluster
+
+Work in progress tasks:
+
+* Designing and implementing pre-commit CI building and testing (to support the link:https://gitlab.com/bsdimp/freebsd-workflow[workflow working group])
+* Designing and implementing use of CI cluster to build release artifacts as release engineering does
+* Testing and merging pull requests in link:https://github.com/freebsd/freebsd-ci/pulls[the FreeBSD-ci repo]
+* Reducing the procedures of CI/test environment setting up for contributors and developers
+* Setting up the CI stage environment and putting the experimental jobs on it
+* Organize the scripts in freebsd-ci repository for preparing to merge to src repository
```

Comment:
"Organizing" for consistency with other bullet points.
s/for preparing to merge/to prepare for merging/ (reads less stilted to me)
"in the ... repository" x 2 (taste)